integration/models/applicationConfig.ts

```diff
@@ -56,16 +57,24 @@ export const applicationConfig = () => {
     addDependency: (name: string, version: string | undefined) => {
       if (version) {
         dependencies.set(name, version);
+      } else if (name.includes('@clerk/')) {
+        // find package folder using the package name
+        const packageNameDirectory = name.split('/').pop()?.replace('clerk-', '');
+        dependencies.set(name, `file:${process.cwd()}/packages/${packageNameDirectory}`);
+      } else {
+        // since we have moved the integrations outside the monorepo we should fallback to * in case
+        // of empty version for non Clerk packages (eg next should install the latest version)
+        dependencies.set(name, '*');
       }
```

Comment:
@nikosdouvlis PTAL this changes as it affect what version we expect to use in our integration tests.

Reply:
I replaced this change with the following actions:
- drop default `*` version
- move the logic to link our packages to local folders in the presets to avoid polluting the `ApplicationConfig` model with Clerk-specific logic.
cc: @nikosdouvlis